video/out/vo_gpu_next.c

```diff
@@ -370,30 +374,19 @@ static void update_overlays(struct vo *vo, struct mp_osd_res res,
         case SUBBITMAP_BGRA:
             ol->mode = PL_OVERLAY_NORMAL;
             ol->repr.alpha = PL_ALPHA_PREMULTIPLIED;
-            // Infer bitmap colorspace from source
-            if (src) {
+            // Infer bitmap primaries and luminance from source
+            if (src)
                 ol->color = src->params.color;
-                if (pl_color_transfer_is_hdr(ol->color.transfer)) {
-                    if (!pl_color_transfer_is_hdr(frame->color.transfer)) {
-                        // Tone mapping targets SDR white
-                        ol->color.hdr = (struct pl_hdr_metadata) {
-                            .max_luma = PL_COLOR_SDR_WHITE,
```

Comment:
Deleting it will bring back problems: #8325

Reply:
Do you have sample? I explicitly changed .transfer to handle this correctly now.

Reply:
Please send a sample, I will check tomorrow.

Reply:
Maybe I didn't check the video with enough brightness. Will check tomorrow.